core/agentbinary/types.go

```diff
@@ -40,3 +42,7 @@ func (v Version) Validate() error {
 	}
 	return nil
 }
+
+func (v Version) String() string {
```

Comment:
Can you add a comment explaining what this done. Be verbose :)

Reply:
Sure thing tom :)